### DISCUSSION TITLE
MathML Options?

### DISCUSSION THREAD

**Discussion by benubois** [rgrove/sanitize#227] [Q&A] [answered]
Hi @rgrove,

Thanks for your work on sanitize! Feedbin makes extensive use of it and I can't imagine it working in a secure way without it.

Dr. Drang's very popular blog is [testing MathML support](https://leancrew.com/all-this/2023/09/testing-mathml/) in RSS readers as an alternative to MathJax. 

I'd like for Feedbin to be able to offer MathML support too, but noticed sanitize can't currently allow it in a safe way.

- Is there any workaround that would allow Feedbin to show MathML securely?
- Could Feedbin sponsor the work on allowing MathML?

Thanks for any help you can give!

**Comment by rgrove** [comment, marked as answer]
Hi @benubois! I'm a Feedbin customer and fan. Cool to hear you're using Sanitize!

I have bad news and good news for you. The bad news is that while I have no plans to stop maintaining Sanitize, I'm not currently interested in expanding its feature set. Adding MathML support would increase the attack surface (and thus the maintenance burden) significantly.

The good news is that [Loofah](https://github.com/flavorjones/loofah) is a fantastic alternative to Sanitize, and it supports MathML! As excited as I am to hear that Feedbin is using Sanitize, it may be worth considering a switch to Loofah if MathML is important to you. 🙂

**Reply by benubois** [reply]
No worries! I completely understand and appreciate your decision.
